src/validators/v1/activities.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * name : validators/v1/activities.js
+ * author : Priyanka Pradeep
+ * Date : 29-Aug-2024
+ * Description : Validations of activities controller
+ */
+
+module.exports = {
+	list: (req) => {
```

Comment:
@priyanka-TL - Activity list will always be with resource id which is mandatory

Reply:
done